### PR TITLE
Add SQL docs

### LINKED ISSUE #2899: Add SQL docs
As database is an important part of IT so, adding the SQL docs will be beneficial

### PR DESCRIPTION
**Pull request by root-0101**
## Related Issue

Fix #2899 

## Description

added the sql docs

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Security enhancement
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have followed the code style guidelines of this project.
- [x] I have checked for any existing open issues that my pull request may address.
- [x] I have ensured that my changes do not break any existing functionality.
- [x] Each contributor is allowed to create a maximum of 4 issues per day. This helps us manage and address issues efficiently.
- [x] I have read the resources for guidance listed below.
- [x] I have followed security best practices in my code changes.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Please read the following resources before submitting your contribution:

- [x] [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [x] [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [x] [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [x] [Markdown Guide](https://www.markdownguide.org/)
- [x] [MDX Documentation](https://mdxjs.com/docs/)
- [x] [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- [x] [Tabs: Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [x] [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [x] [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex) 

## Contribution Guidelines

Thank you for considering contributing to our project! To ensure smooth collaboration and effective contribution management, please adhere to the following guidelines:

### Issue Creation

1. **Limit on Issues:**
   - Each contributor is allowed to create a maximum of **4 issues per day**. This helps us manage and address issues efficiently.

### Contribution Levels

2. **Basic Contributions:**
   - This project is primarily focused on documentation. Most of the setup has been completed, so contributors will generally need to work on basic code tasks, such as writing tests.
   - For these tasks, issues will be assigned the **level1** label.

3. **Acknowledging Hard Work:**
   - If a contributor puts in significant effort on a task, the issue will be upgraded to **level2**. This is our way of recognizing and appreciating extra effort.

4. **Feature Additions and Component Work:**
   - Contributors working on new features or components using JSX/TSX will be assigned a level based on the complexity and quality of their work.
   - The more complex and valuable the contribution, the higher the level assigned.

### Level Definitions

- **Level1:**
  - Tasks are straightforward, such as fixing minor bugs, writing tests, or making simple documentation updates.
- **Level2:**
  - Tasks require more effort, such as addressing complex bugs, improving existing features, or making substantial documentation improvements.
- **Level3:**
  - Tasks are highly complex and involve significant new feature development, major refactoring, or extensive contributions to the project’s core components.

We look forward to your contributions and appreciate your effort in helping us improve the project!

### CODE REVIEW

File: docs/SQL/08-schema-design1.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+In one ticket, we can book multiple seats.
+One seat can be booked in only 1 ticket.
+
+So, the final cardinality between ticket and seat is **one-to-many**
+
+**Example 3:**  Consider a monogamous community. What is the cardinality between husband and wife?
+
+| husband | --- married to --- | wife |
+| ------- | ------------------ | ---- |
+| 1       | -->                | 1    |
```
Comment (Ajay-Dhangar) [inline docs/SQL/08-schema-design1.md:225]: Replace the greater than symbol `>` with `&gt;` and the less than symbol `<` with `&lt;`.

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Please make sure to update your code according to the suggestions.

**Comment by Ajay-Dhangar** [comment]
@root-0101, please complete your task.

**Comment by root-0101** [comment]
done the changes

**Comment by Ajay-Dhangar** [comment]
@root-0101, please review the code you have written for the SQL docs and debug it based on the `mdx/jsx` syntax. The best way to do this is to write code for only one file at a time. If you’re having issues with a particular file, please post the code and I will help you debug it.

**Review by Ajay-Dhangar — APPROVED** [review]
Done

For next time, please be sure to read our contributor resources and guidelines. We've created some custom layouts for the documentation, and it's important to follow those in order to maintain consistency across the site.

**Pull request merged**